compiler/rustc_lint/src/lib.rs

```diff
@@ -336,6 +336,7 @@ fn register_builtins(store: &mut LintStore, no_interleave_lints: bool) {
     store.register_renamed("redundant_semicolon", "redundant_semicolons");
     store.register_renamed("intra_doc_link_resolution_failure", "broken_intra_doc_links");
     store.register_renamed("overlapping_patterns", "overlapping_range_endpoints");
+    store.register_renamed("safe_packed_borrows", "unaligned_references");
```

Comment:
I called this a "rename" now because the lints are fairly similar... but maybe "remove" would be more appropriate?

Reply:
No opinion, seems fine as is.